server/src/databaseAPI/controllers/partyController.js

```diff
@@ -30,6 +30,39 @@ const Party = {
       return res.status(400).send(error);
     }
   },
+  async getOneParty(req, res) {
+    const text = 'SELECT * FROM partiess WHERE id = $1';
+    try {
+      const { rows } = await db.query(text, [req.params.id]);
+      if (!rows[0]) {
+        return res.status(404).send({'message': 'party not found'});
+      }
+      return res.status(200).send(rows[0]);
+    } catch(error) {
+      return res.status(400).send(error)
+    }
+  },
+  async updateParty(req, res) {
+    const findOneQuery = 'SELECT * FROM parties WHERE id=$1';
+    const updateOneQuery =`UPDATE parties
+      SET party_name=$1,party_address=$2,party_logo=$3 returning *`;
+    try {
+      const { rows } = await db.query(findOneQuery, [req.params.id]);
+      if(!rows[0]) {
+        return res.status(404).send({'message': 'party not found'});
+      }
+      const values = [
+        req.body.partyName || rows[0].party_name,
+        req.body.partyAddress || rows[0].party_address,
```

Comment:
It seems you did not consider the last comment I dropped on create party, You can't have camel case and pascal case in your code. Change pascal case `party_name` to camel case `partyName`. I am also wondering why `partyName` instead of just `name`

Reply:
rename the partyName and partyAddress to just name and address